tests/codegen/call_offsetof.cpp

```diff
@@ -0,0 +1,23 @@
+#include "common.h"
+
+namespace bpftrace {
+namespace test {
+namespace codegen {
+
+TEST(codegen, call_offsetof)
+{
+  test("struct Foo { int x; long l; char c; } BEGIN { @x = offsetof(struct Foo, x); @c = offsetof(struct Foo, c); }",
```

Comment:
I'd just leave `@x = offsetof(struct Foo, x)` here, the other offset depends on ABI, so it may not pass on some arches.